aws/components/ecs/state.ftl

```diff
@@ -226,8 +226,10 @@
     [/#if]
 
     [#local serviceId = formatResourceId(AWS_ECS_SERVICE_RESOURCE_TYPE, core.Id)]
+    [#local serviceName = core.FullName]
     [#local taskId = formatResourceId(AWS_ECS_TASK_RESOURCE_TYPE, core.Id) ]
-    [#local taskName = core.Name]
+    [#local taskName = core.FullName]
+    [#-- taskName changed from core.Name --]
```

Comment:
Where is the task name used? This could cause breaking changes with existing deployments. Renaming resources can be dangerous

Reply:
In @createECSTask. I figured it probably would be which is why the comment placeholder, though I thought a mismatched name might be worse

Reply:
Lets leave this change out and if you feel it should be investigated lets raise a new ticket  it for it. We need to see what this does with some live deployments.

Reply:
Cool, done